ICD10GM/load_stage.sql

```diff
@@ -134,6 +137,11 @@ FROM concept_stage c1,
 WHERE c2.concept_code LIKE c1.concept_code || '%'
 	AND c1.concept_code <> c2.concept_code;
 
+/* Due to the similarity of concept_codes in ICD10 and ICD10GM inaccurate mapping can occur in ICD10GM. 
+For these cases necessary mapping was put into concept_relationship_manual table as well as deprecated old one. 
+Thus, these pares should be deleted from concept_relationship_stage on this stage */
+DELETE FROM concept_relationship_stage WHERE concept_code_1 IN (SELECT concept_code_1 FROM concept_relationship_manual) AND vocabulary_id_2 = 'SNOMED';
```

Comment:
do we still need this?
as you already added those relationships as 'D' in concept_relationship_manual

Reply:
Yes, we still need this. Otherwise, there will be alive duplicates for the dead relationship.

Reply:
You are right. This step was useless due to update in manual stages. 
They were put in GoogleDrive and link to them can be found in readme